src/direct-linking/background/storage.test.ts

```diff
@@ -0,0 +1,155 @@
+import indexedDB from 'fake-indexeddb'
+import IDBKeyRange from 'fake-indexeddb/lib/FDBKeyRange'
+import normalize from '../../util/encode-url-for-id'
+
+import Storage from '../../search/search-index-new/storage'
+import { StorageManager } from '../../search/search-index-new/storage/manager'
+// import { AnnotationStorage } from './storage'
+import AnnotationBackround from './'
+
+import * as dummy from './storage.test.data'
```

Comment:
It would probably be better to call this `DATA` to be more descriptive (constant/unchanging test data) and consistent with the other tests that utilize a test data module (actually some are named differently, but most seem to be `DATA`)